test/onnx/expect/TestOperators.test_rrelu.expect

```diff
@@ -0,0 +1,74 @@
+ir_version: 3
+producer_name: "pytorch"
+producer_version: "0.4"
+graph {
+  node {
+    input: "input"
+    output: "1"
+    op_type: "RandomUniformLike"
+    attribute {
+      name: "high"
```

Comment:
high is lower than low?

I think rrelu's signature should be self, lower, upper, training, generator.